website/blog/2024-05-17-whats-next.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: whats-next
+title: What's next for MACI?
+description: Dive into the next MACI protocol changes and improvements.
+authors:
+  name: ctrlc03
+  title: MACI dev
+  url: https://x.com/ctrlc03
+  image_url: https://avatars.githubusercontent.com/u/93448202?v=4
+tags: [voting, security, anonymity]
+---
+
+Minimal Anti Collusion Infrastructure ([MACI](https://github.com/privacy-scaling-explorations/maci)) is a Public Good that allows one to run secure, private, on-chain voting polls.
+
+Given its open source nature, it is not uncommon to develop new features or to have to fix issues based on community feedback. However, something that does not usually happen is external contributors making changes to the core protocol.
+
+Well, this soon will be a reality thanks to a proposal sent by the [3327](https://3327.io/) team. 3327 is collective of 10+ people working on improving blockchain technologies, with a focus on research and engineering.
```

Comment:
```suggestion
Well, this soon will be a reality thanks to a MACI improvement proposal sent by the [3327](https://3327.io/) team. 3327 is collective of 10+ people working on improving blockchain technologies, with a focus on research and engineering. Their development team will be working to implement significant changes to MACI's codebase.
```
Not sure this is the best wording, but clarify that they will be implementing code changes